docs/_documentations/remotedeploy-single.md

```diff
@@ -142,10 +140,10 @@ INFO[0159] Waiting for Codewind PFE to start
 INFO[0159] Codewind is available at: https://codewind-gatekeeper-k55333j0.mycluster-12345-7674b4bd9abbdeea5be228236d5275c9-0001.eu-gb.containers.appdomain.cloud
 ```
 
-Codewind has been sucessfully deployed and available.
+Codewind is successfully deployed and is available.
 
-You can re-run the last command several times and each time a new Codewind deployment will be generated against the existing Keycloak service. A developer must have their own Codewind instance,  however they may also have more than one instances assigned to them through role-based access control which `cwctl` has setup automatically.
+You can re-run the command several times. Each time you re-run the command, a new Codewind deployment is generated against the existing Keycloak service. You must have your own Codewind instance, however, you might also have more that one instance assigned to you through role-based access control which `cwctl` sets up automatically. 
```

Comment:
- The last sentence should be split into two. They are both independent clauses.
- Also replace "which" with "that."

"You must have your own Codewind instance. However, you might also have more that one instance assigned to you through role-based access control that `cwctl` sets up automatically."